pkg/operator/ceph/version/version.go

```diff
@@ -293,3 +295,15 @@ func ValidateCephVersionsBetweenLocalAndExternalClusters(localVersion, externalV
 
 	return nil
 }
+
+func RegisterImageVersion(image string, version CephVersion) {
+	imageToVersionMap[image] = version
+}
+
+func GetImageVersion(image string) (*CephVersion, error) {
+	version, ok := imageToVersionMap[image]
```

Comment:
Since `RegisterImageVersion()` is called from a separate goroutine from `GetImageVersion()`, we do need a mutex protecting the `imageToVersionMap`